<launch>
  <param name="root_path" type="string" value="/data/compare/cliped/" />
  <!-- <param name="root_path" type="string" value="/data/compare/" /> -->

  <!-- <param name="file_name" type="string" value="segment_crf_icp" /> -->
  <!-- <param name="file_name" type="string" value="segment_crf_lio" /> -->
  <!-- <param name="file_name" type="string" value="segment_rb2_icp" /> -->
  <param name="file_name" type="string" value="segment_rb2_lio" />
  <!-- <param name="file_name" type="string" value="segment_hall_icp" /> -->
  <!-- <param name="file_name" type="string" value="segment_hall_lio" /> -->

  <param name="a" type="double" value="-0.0366138" />
  <param name="b" type="double" value="0.000935359" />
  <param name="c" type="double" value="0.999329" />

  <param name="z_lb" type="double" value="-10" />
  <param name="z_ub" type="double" value="10" />

  <node name="ground" pkg="calibration" type="ground" output="screen">
  </node>
</launch>
